setup.py

```diff
@@ -14,7 +13,7 @@
     long_description=readme,
     long_description_content_type="text/x-rst",
     py_modules=["dj_database_url"],
-    install_requires=["Django>3.2"],
```

Comment:
looks like this was accidentally blocking install on still-supported Django 3.2 since #162